.github/workflows/release-vm-console-proxy.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release VM Console Proxy
+
+on:
+  repository_dispatch:
+    types: [release-vm-console-proxy]
+
+jobs:
+  release-vm-console-proxy:
+    name: Release VM Console Proxy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          release_version: ${{ github.event.client_payload.release_version }}
```

Comment:
`release_version` is not an attribute of `action/checkout`: https://github.com/actions/checkout

Reply:
I saw an example of it [1]. It doesn't mean that it's doing checkout code when there is an actual `release_version` passed?